2024/2024-ssl-teams-vs-teams-passing-challenge.adoc

```diff
@@ -0,0 +1,94 @@
+:source-highlighter: highlightjs
+
+= RoboCup 2024 Team vs. Team Passing Challenge
+{docdate}
+:toc:
+:stem: latexmath
+:sectnumlevels: 0
+
+// add icons from fontawesome in a up-to-date version
+ifdef::backend-html5[]
+++++
+<link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.3.1/css/all.css" integrity="sha384-mzrmE5qonljUremFsqc01SB46JvROS7bZs3IO2EmfFsd15uHvIt+Y8vEf7N7fWAU" crossorigin="anonymous">
+++++
+endif::backend-html5[]
+
+:icons: font
+:numbered:
+
+NOTE: References to the male gender in the rules with respect to referees, team
+members, officials, etc., are for simplification and apply to both males and
+females.
+
+== Goal
+
+A key distinguishing characteristic of the Small Size League is its tight integration of software and hardware, which is crucial to create a dynamic, collaborative soccer match. 
+While this goal was pursued in last year's SSL Chip Pass Challenge, the goal of this year's challenge is to utilize last year's work and adapt it to a more realistic situation.
+
+Therefore this year two teams will try to play as many passes as possible while competing against each other. To achieve this, the ruleset used last year has been adapted to the new situation.
+
+Although the teams, which have participated in one of the previous passing challenges, might have a slight advantage, we would like to encourage all teams to participate in this technical challenge.
+
+== Procedure
+
+The technical challenge will be held on a Div B field with a maximum of 4 robots per team.
+
+To level the playing field and to remove the impact of the performance of the opponent team on a teams score, each team will play two games against different contestants of the technical challenge.
```

Comment:
How are the contestants chosen? Might be good to add the general procedure here already.

Reply:
Added a short summary in the current commit